File: binance4j-vision/src/test/java/com/binance4j/spot/SpotTest.java
package com.binance4j.spot;

import com.binance4j.core.client.RestClient;
import com.binance4j.core.dto.CandlestickInterval;
import com.binance4j.core.test.CustomTest;
import com.binance4j.vision.client.VisionSpotClient;

public class SpotTest extends CustomTest {
	VisionSpotClient client = new VisionSpotClient();
	protected final String year = "2022";
	protected final String month = "01";
	protected final String day = "01";
	protected final CandlestickInterval interval = CandlestickInterval.ONE_MINUTE;

	/**
	 * @return the year
	 */
	public String getYear() {
		return year;
	}

	/**
	 * @return the month
	 */
	public String getMonth() {
		return month;
	}

	/**
	 * @return the day
	 */
	public String getDay() {
		return day;
	}

	/**
	 * @return the interval
	 */
	public CandlestickInterval getInterval() {
		return interval;
	}

	@Override
	protected RestClient<?> getClient() {
		return null;
	}
}

File: connectors/src/test/test/com/binance4j/vision/SpotTest.kt
/*
 * MIT License
 *
 * Copyright (c) 2022 Binance4j
 *
 * Permission is hereby granted, free of charge, to any person obtaining a copy
 * of this software and associated documentation files (the "Software"), to deal
 * in the Software without restriction, including without limitation the rights
 * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
 * copies of the Software, and to permit persons to whom the Software is
 * furnished to do so, subject to the following conditions:
 *
 * The above copyright notice and this permission notice shall be included in all
 * copies or substantial portions of the Software.
 *
 * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
 * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
 * FITNESS FOR A PARTICULAR PURPOSE AND NON INFRINGEMENT. IN NO EVENT SHALL THE
 * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 * SOFTWARE.
 */

package com.binance4j.vision

import com.binance4j.core.dto.CandlestickInterval
import com.binance4j.core.test.CustomTest
import com.binance4j.vision.client.VisionSpotClient

open class SpotTest : CustomTest() {
	var client = VisionSpotClient()
	
	/**
	 * @return the year
	 */
	val year = "2022"
	
	/**
	 * @return the month
	 */
	val month = "01"
	
	/**
	 * @return the day
	 */
	val day = "01"
	
	/**
	 * @return the interval
	 */
	val interval = CandlestickInterval.ONE_MINUTE
}